### PR TITLE
194 expand version

### PR DESCRIPTION
**Pull request by lapinlapin**
#### What's this PR do?
Added expand mode for OrderHistory
-Added expand flag
-Fixed font sizes
-Added option to table structure of column
-etc
#### Where should the reviewer start?
views/OrderHistory
#### How should this be manually tested?
http://localhost:8080/#/
#### Screenshots (if appropriate)
image

#### URL
Provide a link to your deploy from http://staging-ui.trusty.apasia.tech:8080/194-expand-version

### CODE REVIEW

File: src/views/OrderHistory/OrderHistorySearch.vue [hunk 1]
```diff
@@ -24,3 +25,9 @@ export default {
   }
 }
 </script>
+<style lang="scss">
+  .order-history-search {
+    transform: scale(1.2, 1.2);
```
Comment (roma219) [inline src/views/OrderHistory/OrderHistorySearch.vue]: Why this scale? Not the best solution to solve the problem with search position on different widths

**Review by roma219 — CHANGES_REQUESTED** [review]
Ok, but the search should be positioned smarter...

File: src/views/OrderHistory/OrderHistorySearch.vue (hunk 1 above)
Comment (lapinlapin) [reply src/views/OrderHistory/OrderHistorySearch.vue]: @roma219 in zeplin icon looks bigger. Therefore, I made a scale

Comment (roma219) [reply src/views/OrderHistory/OrderHistorySearch.vue]: @lapinlapin But why do you scale the whole component for that? We'll need to do in in `Input` component... Two ways here: 1) keep all inputs unified 2) make a special prop for this kind of behaviour. For now i propose just to change the size of the icon inside `SearchInput` component - as all search inputs in widgets should look the same.

Comment (lapinlapin) [reply src/views/OrderHistory/OrderHistorySearch.vue]: I agree 👍

Comment (lapinlapin) [reply src/views/OrderHistory/OrderHistorySearch.vue]: fixed

**Review by roma219 — APPROVED** [review]
👍

**Pull request merged**